modular_skyrat/modules/customization/modules/clothing/under/security.dm

```diff
@@ -52,30 +44,34 @@
 	supports_variations_flags = NONE
 	can_adjust = FALSE
 
-///Police Uniforms
+// Police Uniforms
+///OFFICERS
 /obj/item/clothing/under/rank/security/peacekeeper/sol/cadet
 	name = "sol police cadet uniform"
-	desc = "A light blue shirt with navy pants, perfect for pretending you matter."
+	desc = "A light blue shirt with navy pants, and no other markings. This is essentially no more than a fancy blue suit."
 	icon_state = "policecadetalt"
 
-/obj/item/clothing/under/rank/security/peacekeeper/sol/traffic
-	name = "sol traffic police uniform"
-	desc = "A light blue shirt with navy pants, perfect for standing and shouting at cars."
-	icon_state = "policetrafficalt"
-///OFFICERS
 /obj/item/clothing/under/rank/security/peacekeeper/sol
 	name = "sol police uniform"
-	desc = "A light blue shirt with navy pants, perfect for opressing the Underclasses, like Catgirls."
```

Comment:
Catgirl joke removed. Distressing.

Reply:
Yes, but the new one is much more accurate to the general-uniformed officer IMO. Plus "Catgirls" is too offputting and the word "Underclasses" feels gross

Reply:
Let's get a third opinion from the Cattainer?
@tf-4 Thoughts?

Reply:
Stealth felinid buff. Approved.